checker/dune

```diff
@@ -1,7 +1,25 @@
-(rule (copy %{project_root}/kernel/names.ml names.ml))
-(rule (copy %{project_root}/kernel/names.mli names.mli))
-(rule (copy %{project_root}/kernel/esubst.ml esubst.ml))
-(rule (copy %{project_root}/kernel/esubst.mli esubst.mli))
+(copy_files
+  %{project_root}/kernel/{names,esubst,declarations,environ,constr,term,univ,evar,sorts,uGraph,context}.ml{,i})
+
+(copy_files
+  %{project_root}/kernel/{mod_subst,vars,opaqueproof,conv_oracle,reduction,typeops,inductive,indtypes,declareops,type_errors}.ml{,i})
+
+(copy_files
+  %{project_root}/kernel/{modops,mod_typing,}.ml{,i})
+
+(copy_files
+  %{project_root}/kernel/{cClosure,cPrimitives,csymtable,vconv,vm,uint31,cemitcodes,vmvalues,cbytecodes,cinstr,retroknowledge,copcodes}.ml{,i})
+
+(copy_files
+  %{project_root}/kernel/{cbytegen,clambda,nativeinstr,nativevalues,nativeconv,nativecode,nativelib,nativelibrary,nativelambda}.ml{,i})
+
+(copy_files
+  %{project_root}/kernel/{subtyping,term_typing,safe_typing,entries,cooking}.ml{,i})
+
+; VM stuff
```

Comment:
If you are disabling vm compute do you still to link this?

Reply:
Good question, I thought we could provide a flag that enables VM / native for the checker, for cases when checking takes too much time otherwise. But I didn't expose such a flag yet (the machinery is there, though).

Do you think I should remove the runtime?

Reply:
I guess it is fine to leave it as is for now. Would you refine the VM/Native hooks indeed you could remove from the build a lot of files, which could be important for certification/review purposes, but that can happen later I guess.